persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAO.kt

```diff
@@ -172,7 +172,8 @@ interface ConversationDAO {
     suspend fun updateAllConversationsNotificationDate()
     suspend fun getAllConversations(): Flow<List<ConversationViewEntity>>
     suspend fun getAllConversationDetails(): Flow<List<ConversationViewEntity>>
-    suspend fun getAllProteusTeamConversations(teamId: String): Flow<List<ConversationViewEntity>>
+    suspend fun getAllProteusTeamConversations(teamId: String): List<QualifiedIDEntity>
+    suspend fun getAllProteusTeamConversationsReadyToBeFinalised(teamId: String): List<QualifiedIDEntity>
```

Comment:
This class is getting HUGE.


Same for `ConversationRepository`.

Nothing for this PR to tackle, just raising awareness.